applications/asset_tracker_v2/README.rst

```diff
@@ -299,6 +352,12 @@ You can add the following optional configurations to configure the heap or to pr
 * :option:`CONFIG_PDN_DEFAULTS_OVERRIDE` - Used for manual configuration of the APN. Set the option to ``y`` to override the default PDP context configuration.
 * :option:`CONFIG_PDN_DEFAULT_APN` - Used for manual configuration of the APN. An example is ``apn.example.com``.
 
+The application supports Assisted GNSS.
+To set the source of the AGPS data, set the following options:
```

Comment:
Mismatch between "assisted GNSS" and "AGPS" on this line. I think we can stick to "assisted GPS" and "A-GPS" for now, as that's the only constellation we receive data for

Reply:
Agreed